.github/workflows/continous-integration.yml

```diff
@@ -39,8 +39,8 @@ jobs:
     - name: Install dependencies 🖥
       run: poetry install --no-interaction
 
-    - name: Lint Code 🎎
-      run: make lint
+    - name: pre-commit
```

Comment:
```suggestion
    - name: Lint Code 🎎
```

Reply:
let's even keep CI pipeline untouched and apply changes only in `Makefile`

I think name `lint` is more concise, so better to run `make lint`